### PR TITLE
Add suspended job indicator

### PR DESCRIPTION
**Pull request by e-roux**
Hello,

this brings a symbol ✦ in the prompt in case of suspended job in the background.

### CODE REVIEW

File: pure.zsh [hunk 1]
```diff
@@ -175,7 +175,11 @@ prompt_pure_preprompt_render() {
 		$cleaned_ps1
 	)
 
-	PROMPT="${(j..)ps1}"
+    PROMPT=""
```
Comment (sindresorhus) [inline pure.zsh:178]: Use tab-indentation.

File: pure.zsh [hunk 2]
```diff
@@ -175,7 +175,11 @@ prompt_pure_preprompt_render() {
...
+    if ((${(M)#jobstates:#suspended:*} != 0)); then
+      PROMPT+='%F{$prompt_pure_colors[git:dirty]}✦ '
```
Comment (sindresorhus) [inline pure.zsh:180]: This need a new color instead of reusing one.

File: pure.zsh [hunk 3]
```diff
@@ -175,7 +175,11 @@ prompt_pure_preprompt_render() {
 		$cleaned_ps1
 	)
 
-	PROMPT="${(j..)ps1}"
+    PROMPT=""
+    if ((${(M)#jobstates:#suspended:*} != 0)); then
```
Comment (sindresorhus) [inline pure.zsh:179]: Needs a code comment.

**Review by sindresorhus — CHANGES_REQUESTED** [review]

File: pure.zsh (hunk 2 above)
Comment (mafredri) [reply pure.zsh:180]: This should be moved higher up so that it's part of the `preprompt_parts` array (instead of doing `PROMPT` manipulation here).

**Review by mafredri — APPROVED** [review]
Looks good to me, although I'm not entirely convinced of the position of it (I'd prefer after the path).

But I think it's fine to merge as-is, we can introduce an option to disable and/or move it in case it gets in the way.

**Comment by sindresorhus** [comment]
I think it should be after the path.

**Comment by sindresorhus** [comment]
You also need to add it to https://github.com/sindresorhus/pure#colors

**Comment by e-roux** [comment]
After the path, it is common to have the git information. I fear the region will be overloaded with symbols, especially when on git dirty status therefor I tend to have it at the very beginning.

File: readme.md [hunk 4]
```diff
@@ -156,6 +156,7 @@ Colors can be changed by using [`zstyle`](http://zsh.sourceforge.net/Doc/Release
 - `prompt:error` (red) - The `PURE_PROMPT_SYMBOL` when the previous command has *failed*.
 - `prompt:success` (magenta) - The `PURE_PROMPT_SYMBOL` when the previous command has *succeeded*.
 - `prompt:continuation` (242) - The color for showing the state of the parser in the continuation prompt (PS2). It's the pink part in [this screenshot](https://user-images.githubusercontent.com/147409/70068574-ebc74800-15f8-11ea-84c0-8b94a4b57ff4.png), it appears in the same spot as `virtualenv`. You could for example matching both colors so that Pure has a uniform look.
+- `suspended_jobs` (red) - The four pointed star showing jobs runs in the background.
```
Comment (sindresorhus) [inline readme.md:159]:
```suggestion
- `suspended_jobs` (red) - The `✦` symbol indicates that jobs are running in the background.
```

**Pull request merged**